config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
   root to: 'home#show'
   resources :users, only: [:new, :create, :show, :update]
+  post :sms, to: 'sms/users#update'
```

Comment:
is there something else we want to call this route or does twilio need it to be "sms"?